### ISSUE TITLE
Add description of properties to xzy file

### ISSUE THREAD

**Issue by nathanfranklin** [simphony/AViz#37]
Add description to end of file so that Aviz knows additional information about the parameters.

This is related to https://github.com/simphony/simphony-aviz/issues/8

The file and new ending will look like the following:
```
4
#XZYfile
X0 1.9265350878690097e+00 1.2235141067428751e+01 -9.0000000000000000e+00 1.0 1.0
X0 1.9265350878690097e+00 1.2235141067428751e+01 -5.0000000000000000e+00 1.0 1.0
properties:
        -name: CUBA.RADIUS
        -name: CUBA.MASS
```

**Comment by nathanfranklin** [comment]
@jeremy-rutman, @simphony/fraunhofer-iwm

**Comment by ahashibon** [comment]
more specifically: add visualisation metadata in YAML format to the end of the "standard" xyz format. This will not affect compatibility with (most) other xyz-tools, and allow us to quickly convey the infomation to the visualisation client. Now focus on the nomenclature of the properties in the AViz xyz file.

**Comment by nathanfranklin** [comment]
this is closed by #41
